src/app/workorder/workorder.js

```diff
@@ -191,6 +193,8 @@ angular.module('app.workorder', [
   self.result = result;
 
   mediator.subscribeForScope('wfm:workorder:updated', $scope, function(workorder) {
+    //remove generated angular variables
+    workorder = JSON.parse(angular.toJson(workorder));
```

Comment:
Same as above, this is published in the https://github.com/feedhenry-raincatcher/raincatcher-workorder/blob/master/lib/angular/directive.js#L150 module.

It would be better to fix it there.